GannettDigital/dmpt-candidate-exercise
# Feature :  Inline Advertising Units

**Feature Requirements**
Add inline ad units within the article body. They must conform to the following criteria

- Inline ads must request/render only 300x250's (see https://developers.google.com/doubleclick-gpt/reference#googletag.defineSlot for gpt documentation)
- First ad must be at least 200 words from the top of the article
- Each subsequent ad must be at least 100 words from the last inline ad
- Ads must not be adjacent to <aside> or <figure> elements
- Ads must be between two paragraphs. The only exception is when an available spot precedes a <h2>
- Inline ads must lazy load when they come into view
- Inline ads must be centered and have a label above them that reads "Advertisement - Read More"
- Must work on IE 11

**Testing Steps**

- Download repository
- Switch to feature_inline_ads branch
- Open index.html
- Scroll to see inline ad units within body of the article

**Expectations**

- Candidate will review this pull request commenting on any issues they see, or areas where they can suggest improvement in code quality
- Candidate shouldn't spend longer than an hour doing this. We are not looking for perfection, but how a candidate can clearly communicate issues they see, as well as guide the developer who opened this pull request in their approach to solving the above request.
